resources/docs/migrations.rst

```diff
@@ -129,5 +129,8 @@ Option                Description
 ``--seeder``          The class name of the seeder.
 ====================  =======================================================================
 
+Next chapter: [Usage](usage.rst)<br />
```

Comment:
These links don't work either and you can't use HTML in rst. Let's use something like:

```rst
« `Usage <usage.rst>`_ | `Configuration <configuration.rst>`_ »
```